features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesPresenter.kt

```diff
@@ -216,7 +216,7 @@ class MessagesPresenter @AssistedInject constructor(
             TimelineItemAction.ViewSource -> handleShowDebugInfoAction(targetEvent)
             TimelineItemAction.Forward -> handleForwardAction(targetEvent)
             TimelineItemAction.ReportContent -> handleReportAction(targetEvent)
-            TimelineItemAction.EndPoll -> handleEndPollAction(targetEvent)
+            TimelineItemAction.EndPoll -> targetEvent.eventId?.let { timelineState.eventSink(TimelineEvents.PollEndClicked(it)) }
```

Comment:
Not a big deal, but I was happy with all the `handle*` fun :)

Reply:
You'll soon be happy again